istioctl/cmd/istiodconfig.go

```diff
@@ -155,6 +156,17 @@ func (ga *getAllLogLevelsState) run(out io.Writer) error {
 		}
 		_, err = out.Write(outputBytes)
 		return err
+	case "yaml":
```

Comment:
This is duplicate code with JSON output, you can just do something like `case "json", "yaml"` and if it's yaml, do the rest of the logic.